content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -193,7 +193,7 @@ This admission controller will deny exec and attach commands to pods that run wi
 allow host access.  This includes pods that run as privileged, have access to the host IPC namespace, and
 have access to the host PID namespace.
 
-The DenyEscalatingExec admission plugin is deprecated and will be removed in v1.21.
+The DenyEscalatingExec admission plugin is deprecated.
```

Comment:
@reylejano there were conflicts between master and dev-1.21, i've chosen to fix them by following your recent change from master here.